### ISSUE TITLE
Accessibility: Plugin byline color contrast failure on plugin single

### ISSUE THREAD

**Issue by amberhinds** [WordPress/wordpress.org#261]
On the plugin single page ([Gutenberg plugin test page](https://test.wordpress.org/plugins/gutenberg/)), the byline for the plugin is  #78848f (light gray) on a white background which has a contrast ratio of 3.82:1. 

<img width="244" alt="Screenshot of byline" src="https://github.com/WordPress/wordpress.org/assets/13153456/57969492-24db-47bd-84d3-e315c4e2677c">

<img width="250" alt="Screenshot of color code and contrast check in Chrome dev tools" src="https://github.com/WordPress/wordpress.org/assets/13153456/22a71eb7-70d3-4678-b0c9-c6e744d0af41">

This should pass contrast by at least 4.5:1.

**Comment by ryelle** [comment]
This should use `var(--wp--preset--color--charcoal-4)`, #656a71, which has a 5.4:1 contrast.

**Comment by StevenDufresne** [comment]
Updated in https://meta.trac.wordpress.org/changeset/13545.
